broker-daemon/broker-rpc/orderbook-service/get-orderbook.js

```diff
@@ -25,6 +35,9 @@ async function getOrderbook ({ params, logger, orderbooks }, { GetOrderbookRespo
     const datetime = nano.toISOString(currentTime)
     const orders = await orderbook.all()
 
+    // limit is passed as an integer with default protobuf value of '0', so '0' implies limit was omitted
+    const limit = (params.limit === '0' || params.limit === undefined) ? DEFAULT_LIMIT : Big(params.limit)
```

Comment:
Good question. Quick answer is no, I wouldn't expect `params.limit` to be undefined.

I defined `limit` to be consistent with the implementation of the `limit` param in `getTrades` (https://github.com/sparkswap/broker/blob/master/broker-daemon/broker-rpc/info-service/get-trades.js#L37). Since both `getTrades` and `getOrderbook` are implementations for grpc methods, I think it's safe to assume that `limit` won't be undefined since the default protobuf value will be `'0'`, but let me know if you think it's valuable to keep the check for undefined.

I've updated `getOrderbook` to remove the check for `params.limit === undefined`.

Question: Would it be appropriate to also update `getTrades` to remove `params.limit === undefined` and the corresponding tests, or does that muddy the scope of this PR?

Reply:
@bradleystachurski figured that we would do something like that XD. Feel free to remove it in this PR. I'll do a re-review and we can go from there.